azext_edge/edge/providers/orchestration/resources/dataflows.py

```diff
@@ -21,16 +24,118 @@
         DataflowProfileOperations,
     )
 
+console = Console()
 
 class DataFlowProfiles(Queryable):
     def __init__(self, cmd):
         super().__init__(cmd=cmd)
+        self.instances = Instances(cmd=cmd)
         self.iotops_mgmt_client = get_iotops_mgmt_client(
```

Comment:
`Instances` already has an `iotops_mgmt_client` reference, so that can be re-used rather than initializing a new one.